ocs_ci/utility/managedservice.py

```diff
@@ -92,29 +92,45 @@ def remove_header_footer_from_key(key):
     return "".join(key_lines)
 
 
-def get_storage_provider_endpoint(cluster):
+def get_storage_provider_endpoint(wait=False, timeout=1080):
     """
-    Get get_storage_provider_endpoint
+    Get storage provider endpoint from storage cluster.
 
     Args:
-        cluster (str): cluster name
+        wait (bool): If true then wait for the value to be available for
+            number of seconds defined in timeout parameter. If false then
+            trye to return value only once.
+        timeout (int): Number of seconds to wait for the value
 
     Returns:
         str: value of storage provider endpoint
 
     """
+    # TODO(fbalak): use following command when https://github.com/red-hat-storage/ocs-ci/pull/5541
+    # is merged:
+    # config.switch_to_provider()
+    for i, cluster in enumerate(config.clusters):
+        if cluster.ENV_DATA["cluster_type"] == "provider":
+            provider_id = i
+            break
```

Comment:
What if you don't find the provider?

Reply:
This will be replaced with a function from config module that raises ClusterNotFoundException if provider not found.